puppet/zulip/files/cron.d/soft-deactivate-users

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+# Cron job to soft deactivate long term idle users weekly.
+0 5 * * 7 zulip /home/zulip/deployments/current/manage.py soft_activate_deactivate_users -d --inactive-for 30
```

Comment:
You also need to add this file to the `puppet` manifests, or it won't actually run.  I think you want it in `puppet/zulip/manifests/app_frontend.pp`.  

Also, to make sure this code doesn't run on staging servers, we should add a check for `settings.STAGING` in the management command, and have it do nothing if that's `True`.